glooshot/docs/tutorial/bookinfo_tutorial.md

```diff
@@ -0,0 +1,265 @@
+As you can see, scrape configs that are both insightful and resource-efficient can be quite complicated.
+Additionally, managing Prometheus configs for multiple scrape targets can be difficult.
+
+Fortunately, Supergloo provides a powerful utility for configuring your Prometheus instance in such a way that is
+appropriate for your chosen service mesh.
+
+By default, `glooshot init` deploys an instance of Prometheus (this can be disabled).
+For best results, you should configure this instance of Prometheus with the metrics that are relevant to your particular service mesh.
+We will use the `supergloo set mesh stats` utility for this.
+
+```bash
+supergloo set mesh stats \
+    --target-mesh glooshot.istio-istio-system \
+    --prometheus-configmap glooshot.glooshot-prometheus-server
+```
+
+Note that we just had to tell Supergloo where to find the mesh description and where to find the config map that we want to update.
+Supergloo knows which metrics are appropriate for the target mesh and sets these on the active prometheus config map.
+You can find more details on setting Prometheus configurations with Supergloo [here](https://supergloo.solo.io/tutorials/istio/tutorials-3-prometheus-metrics/).
+
+#### Deploy the bookinfo app
+
+- Now deploy the bookinfo app to the default namespace
+```bash
+kubectl apply -f https://raw.githubusercontent.com/solo-io/glooshot/master/examples/bookinfo/bookinfo.yaml
+```
+
+- Verify that the app is ready.
+  - When the pods in the `default` namespace are ready, we can start testing our app
+```bash
+kubectl get pods -n default -w
+```
+
+- Let's access the landing page of our app
+  - Execute the command below.
+  - Navigate to http://localhost:9080/productpage?u=normal in your browser.
+  - You should see a book description, reviews, and ratings - each provided by their respective services.
+  - Reload the page a few times, notice that the ratings section changes. Sometimes there are no stars, other times red or black stars appear. This is because Istio is load balancing across the four versions of the reviews service. Each reviews service renders the ratings data in a slightly different way.
+```bash
+kubectl port-forward -n default deployment/productpage-v1 9080
+```
+
+- Let's use Supergloo to modify Istio's configuration such that all reviews requests are routed to the version of the service that has red stars - and an **unknown vulnerability!**
+  - Execute the command below
+  - Now when you refresh the page, the stars should always be red.
+```bash
+supergloo apply routingrule trafficshifting \
+    --namespace glooshot \
+    --name reviews-v4 \
+    --dest-upstreams glooshot.default-reviews-9080 \
+    --target-mesh glooshot.istio-istio-system \
+    --destination glooshot.default-reviews-v4-9080:1
```

Comment:
Curious, why is this v4? in Istio, the red version is v3

Reply:
I think I see as I went farther down the tutorial: v4 is our version with the cascading error. v3 is still v3 from Istio?

Reply:
yeah, v3 was not changed, we made v4 with the new error. Maybe I should change the name to make it more obvious, like "reviews-vulnerable" instead of "reviews-v4"